hugegraph-server/hugegraph-core/src/main/java/org/apache/hugegraph/backend/id/IdGenerator.java

```diff
@@ -196,11 +196,11 @@ public String toString() {
         }
     }
 
-    public static final class LongId extends Number implements Id {
+    public static class LongId extends Number implements Id {
 
         private static final long serialVersionUID = -7732461469037400190L;
 
-        private final long id;
+        protected Long id;
```

Comment:
no, because primitive type `long` can't be set to null by hand. We need a data structure that can be manually set to null **so that it can be GCed immediately** to free up memory. Primitive long can only be GCed along with the original object.
<img width="321" alt="image" src="https://github.com/user-attachments/assets/4661d1ab-6b8a-474c-9843-949466ef1e2e">